src/metabase/public_settings/premium_features.clj

```diff
@@ -323,7 +329,7 @@
 ;; 'enhancements'.
 (define-premium-feature ^:deprecated enable-enhancements?
   "Should we various other enhancements, e.g. NativeQuerySnippet collection permissions?"
-  nil
+  :enhancements
```

Comment:
This is just to ensure the `premium-features` set includes all of our enterprise directory names; doesn't change any behavior